src/components/AllNotesFeed.js

```diff
@@ -0,0 +1,78 @@
+import { useModal } from "contexts/ModelContext";
+import { useNote } from "contexts/NoteContext";
+import { useState } from "react";
+import noNote from "toolkit/assets/no-data.svg";
+import { searchNote } from "toolkit/utils";
+import { Note } from "./Note";
+
+export const AllNotesFeed = () => {
+  const { notes } = useNote();
+  const [searchInputText, setSearchInputText] = useState("");
+  const [searchText, setSearchText] = useState("");
+  const { setModalOpen } = useModal();
+
+  const filteredNotes = searchNote(notes.allNotes, searchText);
+  return (
+    <>
+      <div className="notefeed-content-header row-flex p-2 m-t-3">
+        <label className="field note-searchfield w-80p" htmlFor="search-text">
+          <span className="search-icon cursor p-h-1">
+            <i className="fa fa-search" aria-hidden="true"></i>
+          </span>
+          <input
+            type="text"
+            className="input search-nav reset-ip p-05"
+            placeholder="Search here"
+            value={searchInputText}
+            id="search-text"
+            onChange={(e) => setSearchInputText(e.target.value)}
+            onKeyDown={(e) => (e.key === "Enter" ? setSearchText(searchInputText) : "")}
```

Comment:
suggestion:
you can use && shorting if an else expression is not required ex:
onKeyDown={(e) => e.key === "Enter" && setSearchText(searchInputText)}

Reply:
Oh didn't thought of it. Thanks made the said changes!